handbook/business-operations/README.md

```diff
@@ -125,6 +125,16 @@ Certain new team members, especially in go-to-market (GTM) roles, will need paid
 > **Warning:** Do NOT buy LinkedIn Recruiter. AEs and SDRs should use their personal Brex card to purchase the monthly [Core Sales Navigator](https://business.linkedin.com/sales-solutions/compare-plans) plan. Fleet does not use a company wide Sales Navigator account. The goal of Sales Navigator is to access to profile views and data, not InMail.  Fleet does not send InMail. 
 
 
+### Report on status of customer financial actions
+This reporting is performed to update the status of open or upcoming customer actions regarding the financial health of the opportunity. To complete the report:
+- Go to this [report folder](https://fleetdm.lightning.force.com/lightning/r/Folder/00lUG000000DstpYAC/view?queryScope=userFolders) in SFDC. The three reports will provide the data used in the report.
+- Post in the #g-sales slack channel in the following format, using the data from the SFDC reports:
+> Weekly revenue report - [@ mention CRO and CEO]
```

Comment:
Replace lines 132 - 135 with a code block that is easier to copy and paste into Slack, see below

```
Weekly revenue report - [@`todo: CRO` and `@todo: CEO`]
- Number accounts with outstanding balances = `todo`
- Number of customers awaiting invoices = `todo`
- Number of past-due renewals = `todo`
```